modules/webauthorprofile/lib/webauthorprofile_webinterface.py

```diff
@@ -398,13 +398,19 @@ def index(self, req, form):
         profile_page = WebProfilePage("profile", webapi.get_longest_name_from_pid(self.person_id))
         profile_page.add_profile_menu(menu)
 
+        if 'form_email' in pinfo:
+            gFormEmail = pinfo['form_email']
```

Comment:
Yep basically my question was: is it going to work properly for authenticate users? Is the email of the authenticated user going to be considered at all?